src/biokbase/narrative/clients.py

```diff
@@ -19,20 +19,20 @@ def reset():
 def __init_client(client_name, token=None):
     if client_name == "workspace":
         c = Workspace(URLS.workspace, token=token)
-    elif client_name == "narrative_method_store":
-        c = NarrativeMethodStore(URLS.narrative_method_store, token=token)
-    elif client_name == "user_and_job_state":
-        c = UserAndJobState(URLS.user_and_job_state, token=token)
-    elif client_name == "catalog":
-        c = Catalog(URLS.catalog, token=token)
-    elif client_name == "service" or client_name == "service_wizard":
-        c = ServiceClient(URLS.service_wizard, use_url_lookup=True, token=token)
     elif (
         client_name == "execution_engine2"
         or client_name == "execution_engine"
         or client_name == "job_service"
     ):
         c = execution_engine2(URLS.execution_engine2, token=token)
+    elif client_name == "narrative_method_store":
+        c = NarrativeMethodStore(URLS.narrative_method_store, token=token)
+    elif client_name == "service" or client_name == "service_wizard":
+        c = ServiceClient(URLS.service_wizard, use_url_lookup=True, token=token)
+    elif client_name == "catalog":
+        c = Catalog(URLS.catalog, token=token)
+    elif client_name == "user_and_job_state":
+        c = UserAndJobState(URLS.user_and_job_state, token=token)
```

Comment:
there's no reference to this client name anywhere -- does it even need to be in here any more?

Reply:
Yeah, UJS is defunct, I think. Might be worth asking @MrCreosote or @scanon if it's even still deployed.